sips/minutes/_posts/2016-08-20-sip-20th-september-minutes.md

```diff
@@ -0,0 +1,238 @@
+yet to be seen its feasibility in practice. Martin suggests that as the
+Committee should analyze spores as it is right now. Heather emphasizes that the
+current design is not very useful, so she asks whether we should keep it as such
+or try to improve it.
+
+Martin encourages to improve the spores proposal, but suggests to close the
+proposal because it's not yet ready. Heather proposes to give the proposal
+another iteration to update and improve it.
+
+**Outcome**: The spores proposal will have another iteration in two months
+(November). By then, Heather is asked to update the proposal and try to find out
+a way to provide more thorough checks (spores transitivity).
+
+### Discussion of SIP-26: Unsigned Integer Data Types
+
+Sébastien introduces his updates on the proposal. He's tried hard to remove the
+performance regression in his implementation, but he hasn't found a way. He
+describes that the implementation is still 6% slower because of performance hits
+in the hashCode and equals implementation of a case class. To work around this
+issue, two things are required: 
+
+  * Changing the underlying implementation of byte and short ints.
+  * Making the super class of unsigned integer extend *java.lang.Number*
+
+The second option is not feasible because some numbers are AnyVals, and they can
+only extend objects. Working around this in the backend is, in Sébastien's
+opinion, not an exciting adventure to embark on: a lot of patches and quirky
+fixes are required in the compiler. Sébastien, recognizing his unability to fix
+the issue, recommends to reject the proposal.
+
+Josh needs to leave. Eugene wonders if these problems are only JVM-specific.
+Sébastien replies that both yes and no, and he confirms that unsigned integers
+will be implemented in Scalajs alone, so the implementation won't be
+platform-independent. Eugene is interested in knowing if there will be any code
+duplication in the implementation, and Sébastien doesn't think so, since Scala
+Native implements unsigned integer in a different way.
+
+**Outcome**: The proposal is rejected unanimously.
+
+### Discussion of SIP-27: Trailing commas
+
+Jorge welcomes the guest Dale Wijnand, author of the trailing commas proposal.
+Jorge also expects the discussion about the proposal to shed some light on its
+future status, be it rejected, accepted or given another iteration.
+
+Eugene introduces the updates on the proposal. Dale has asked several questions
+to the committee, the most important being if they would accept trailing commas
+only for multi-line comma separated lines. The Committee focuses on this main
+question, although some other questions about the trailing commas specification
+are on the table.
+
+Sébastien points out that it's a good compromise, he personally likes it. He
+compares trailing commas to the semantics of semicolons in Scala. Martin is not
+convinced but the specialized version and makes the points that trailing commas
```

Comment:
I'm so confused by GitHub's review comments not collapsing, but this is still pending.